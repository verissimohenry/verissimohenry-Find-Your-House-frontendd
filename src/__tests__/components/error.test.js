/* eslint-disable no-unused-vars */
import React from 'react';
import renderer from 'react-test-renderer';
import error from '../../components/Error';

it('renders correctly', () => {
  const error = renderer.create(<error />).toJSON();
  expect(error).toMatchSnapshot();
});
