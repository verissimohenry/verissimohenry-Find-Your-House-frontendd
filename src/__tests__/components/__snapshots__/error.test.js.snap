// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `<error />`;

exports[`renders error component correctly 1`] = `<error />`;
